ros_ws/src/arudino/motor/motor.ino

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file motor.ino
+ * @brief Controls the Motors
+ */
+
+#include <ros.h>
+#include <std_msgs/Float64.h>
+
+#define pi 3.14
+
+/*************
+ * Constants *
+ *************/
+ //Radians Per Second at full Throtall
+const float MAX_RAD_SEC = (8*pi);
+
+//Pins used to controll the Motor
+const int FORWARD_PWM_PIN = 3;
+const int REVERSE_PWM_PIN = 4;
+const int ENABLE_PIN = 5;
+ 
+/************************
+ * Forward Declerations *
+ ************************/
+void VelocityCallback(const std_msgs::Float64 &msg);
+
+//ROS node handle
+ros::NodeHandle nodeHandle;
+
+//ROS Subscriber for the Velocity Message
+ros::Subscriber<std_msgs::Float64> 
+     velocitySub("wheel_velocity", &VelocityCallback);
```

Comment:
@islamelnabarawy Any suggestions on how we should handle constants like this in the arduino files?  Including the common/constants.h becomes a mess when working with the arduino IDE as far as I can tell you have to include it in your sketch or in your arduino libraries folder.